syntax = "proto3";

package message;

message Person {
  string name = 1;
  int32 age = 2;
}

message Message {
  string myString = 1;
  int32 myInt = 2;
  Person myPerson = 3;
}